helm/thingsboard/templates/ingress.yaml

```diff
@@ -0,0 +1,126 @@
+#
```

Comment:
One question: in EVP2 it'll be advisable that users not talk directly with the TB API, and only do so via our APIs (this is to avoid inconsistencies, etc.). I'm wondering if it'd make sense to NOT expose the API endpoints on the ingress. That said, if we want to support the UI it's probably hard to avoid exposing the APIs, although maybe we could restrict the exposed APIs to those used by the UI.

We may need more conversation to clarify this with Product, so don't block the PR for that, I just wanted to float this point.

Reply:
I'm disabling by default:
https://github.com/midokura/thingsboard-ce-k8s/blob/helm/helm/thingsboard/values.yaml#L38

Reply:
I guess that is good for now, later on we can add enablement flags for each back-end path.